### PR TITLE
[W14-1] BrainTrain

### PR DESCRIPTION
**Pull request by eugenefdw**
@digipie @jeraldtsy @lallanachang @halcyoneee

### CODE REVIEW

**Comment by DigiPie** [comment]
Looks good! Thanks for the effort @eugenefdw ! 😃

**Comment by kylase-learning** [comment]
## Feedback (Week 7; v1.1)

### Github Project Page
- [ ] Update Project description
- [ ] Update Link to Project Page

### README
- [x] UI Mockup updated in project website
- [ ] Make the project website look like a product home page (aka remove the top nav bar)
- [x] AB4 acknowledgement
- [x] Travis badge updated to own fork's
- [x] Coveralls badge updated to own fork's

### [Project Page](https://cs2103-ay1819s2-w14-1.github.io/main/)
- [ ] Update to remove AB4 reference
- [ ] Travis badge updated to own fork's
- [ ] Coveralls badge updated to own fork's

### About Us page
- [x] Updated with photos
- [x] Roles and components assigned

UG/DG feedback will be provided to you in the documents.

File: build.gradle [hunk 1]
```diff
@@ -156,7 +158,7 @@ test {
         }
 
         if (runGuiTests) {
-            test.include 'systemtests/**'
+        //    test.include 'systemtests/**'
```
Comment (kylase-learning) [inline build.gradle]: I assume that you are commenting this out temporarily?

File: src/main/java/seedu/address/MainApp.java [hunk 2]
```diff
@@ -15,16 +16,18 @@
...
-import seedu.address.model.Model;
-import seedu.address.model.ModelManager;
-import seedu.address.model.ReadOnlyAddressBook;
+import seedu.address.model.Lessons;
 import seedu.address.model.ReadOnlyUserPrefs;
 import seedu.address.model.UserPrefs;
-import seedu.address.model.util.SampleDataUtil;
-import seedu.address.storage.AddressBookStorage;
-import seedu.address.storage.JsonAddressBookStorage;
+import seedu.address.model.modelmanager.management.ManagementModel;
```
Comment (kylase-learning) [inline src/main/java/seedu/address/MainApp.java]: I recalled that some of you mentioned that it is difficult to conform to the ordering imposed by the style checker. 

I think (high probability) that the ordering is in ascending alphabetical order partitioned by each period, i.e. it is sorted by `package` them `subpackage` then `subsubpackage` (in the form of `package.subpackage.subsubpackage`).

There is a rational behind this ordering as if you want to look through the imports it is easier to skim through via this order. 

Hope this helps!

File: build.gradle (hunk 1 above)
Comment (halcyoneee) [reply build.gradle]: Yes, this will be up again when we have an actually working bottom panel GUI

**Pull request closed without merging**